CONTRIBUTING.md

```diff
@@ -92,3 +103,24 @@ branch are tagged into a release monthly.
 - If your PR refers to or fixes an issue, be sure to add `refs #XXX` or `fixes #XXX` to the PR description. Replacing `XXX` with the respective issue number. See more about [Linking a pull request to an issue
   ](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
 - Be sure to fill the PR Template accordingly.
+
+## Troubleshooting
+
+### No PostHog API key, so analytics won't work
```

Comment:
```suggestion
### EADDRINUSE: address already in use :::3030
```

The PostHog bit is actually just a console.log, not an error. It's unrelated to the issue you were having.

Reply:
@matt-aitken Fixed.